go-controller/pkg/node/ovn_test.go

```diff
@@ -31,10 +34,15 @@ func NewFakeOVNNode(fexec *ovntest.FakeExec) *FakeOVNNode {
 	}
 }
 
-func (o *FakeOVNNode) start(ctx *cli.Context, objects ...runtime.Object) {
+func (o *FakeOVNNode) start(ctx *cli.Context, egressObjects runtime.Object, objects ...runtime.Object) {
```

Comment:
Why is this actually even needed on the node side? You don't have any code touching egress firewall on the node side. Just initialize an empty `egressfirewallfake.NewSimpleClientset()` and restore the function signature.

Reply:
done